docs/sign-up-and-onboard-to-techpass.md

```diff
@@ -28,7 +28,7 @@ Following are the prerequisites to sign up for TechPass and complete the onboard
 
 3. Indicate if you want to onboard your Internet Device to SEED and then select **I'm not a robot**.
 
-   !> To access services such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED.
+   !> - To access services such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED.<br> - If your email is not eligible for SEED (e.g. is not a WOG account), you will not see the option to onboard to SEED.
```

Comment:
```suggestion
   !> - To access services such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED.<br> - If your email is not eligible for SEED (e.g. a schools.gov.sg), you will not see the option to onboard to SEED.
```

honestly not certain if this statement will bring more qns or it should be more intuitive from the system. we can leave it in first